### PR TITLE
docs: user docs for sensitive data feature

### PR DESCRIPTION
**Pull request by LiuJi-Jim**
### Screenshots

10 1 1 234_4000_docs_administration_sensitive-data(1600_900) (1)

10 1 1 234_4000_docs_administration_sensitive-data(1600_900) (2)

10 1 1 234_4000_docs_administration_sensitive-data_(1600_900)

10 1 1 234_4000_docs_administration_sensitive-data(1600_900) (4)

10 1 1 234_4000_pricing(1600_900)

### CODE REVIEW

**Comment by LiuJi-Jim** [comment]
Tips: the preview site will not show the updated pricing page, since we didn't bump the VERSION.

File: content/docs/administration/sensitive-data.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: Sensitive Data
+---
+
+<hint-block type="info">
+
+This feature is only available in the Enterprise plan.
+
+</hint-block>
+
+DBAs and workspace owners can mark multiple database table columns as "Sensitive".
```
Comment (tianzhou) [inline content/docs/administration/sensitive-data.md:11]:
```suggestion
Workspace Owners and DBAs can mark table columns as `Sensitive`.
```

File: content/docs/administration/sensitive-data.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+See the following example, the "salary" column is marked as "Sensitive".
```
Comment (tianzhou) [inline content/docs/administration/sensitive-data.md:13]:
```suggestion
See the following example, the "salary" column is marked as `Sensitive`.
```

File: content/docs/administration/sensitive-data.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+![setting](/static/docs/administration/sensitive-data/sensitive-data-setting.webp)
+
+You may also manage all "Sensitive" columns in "Settings / Workspace / Sensitive Data".
+
+![workspace setting](/static/docs/administration/sensitive-data/sensitive-data-workspace-setting.webp)
+
+Then, the query result of "Sensitive" columns from the [SQL Editor](/docs/sql-editor/overview) will be displayed as "\*\*\*\*\*\*".
```
Comment (tianzhou) [inline content/docs/administration/sensitive-data.md:21]:
```suggestion
Then, the query result of `Sensitive` columns from the [SQL Editor](/docs/sql-editor/overview) will be masked as "\*\*\*\*\*\*".
```

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/docs/administration/sensitive-data.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+</hint-block>
+
+DBAs and workspace owners can mark multiple database table columns as "Sensitive".
+
+See the following example, the "salary" column is marked as "Sensitive".
+
+![setting](/static/docs/administration/sensitive-data/sensitive-data-setting.webp)
+
+You may also manage all "Sensitive" columns in "Settings / Workspace / Sensitive Data".
```
Comment (adela-bytebase) [inline content/docs/administration/sensitive-data.md:17]: "Settings / Workspace / Sensitive Data" -> **Settings** / **Workspace** /  **Sensitive Data**

File: content/docs/administration/sensitive-data.md (hunk 2 above)
Comment (adela-bytebase) [inline content/docs/administration/sensitive-data.md:13]: "salary" -> `salary`

**Pull request merged**